Stack/Opc.Ua.Core/Stack/Transport/ITransportListenerCallback.cs

```diff
@@ -46,5 +48,29 @@ public interface ITransportListenerCallback
         /// <returns>The response to return over the secure channel.</returns>
         /// <seealso cref="BeginProcessRequest" />
         IServiceResponse EndProcessRequest(IAsyncResult result);
+
+        /// <summary>
```

Comment:
these interfaces seem to be duplicates for IServerBase and ITransportListenerChannel, would it be possible to seperate them out in a IAuditEventCallback interface and use it instead in both places, so these interfaces remain unchanged, not causing a breaking change?